MapboxNavigation/TurnArrowView.swift

```diff
@@ -93,6 +93,7 @@ public class TurnArrowView: UIView {
                 flip = true
             case .uTurn:
                 StyleKitArrows.drawArrow180(scale: scale)
+                flip = [.left, .slightLeft, .sharpLeft].contains(ManeuverDirection(angle: angle))
```

Comment:
@bsudekum pointed out that `maneuverDirection` in this case is just `uTurn`, but we want to know the angle underlying that classification. So it’s appropriate to look at `angle`, but I think this is overkill: as long as `angle` is relative (with 0° meaning straight ahead) as opposed to absolute (0° meaning due north), we would flip the arrow if and only if the angle is negative.